### PR TITLE
Fix PNWDataStore too many files open error

### PR DESCRIPTION
**Pull request by niyiyu**
This PR fix the problem that when a day-long mSEED contains too many gaps, the file system could not handle them well.

### CODE REVIEW

**Comment by niyiyu** [comment]
Something haven't been implemented are that: 
- When merging the streams, should we fill np.nan, or 0
- Should we skip those mSEED with too many gaps? (gaps more than 100?)
Maybe @mdenolle have thoughts?

**Comment by mdenolle** [comment]
Oh great! Glad that it's not a system issue.
1. filling with zeros might be best, provided that we apply tapers to all segments of the streams.
2. past 10 gaps, i would skip.

**Comment by niyiyu** [comment]
Thanks @mdenolle

I add a 3% taper to each of the segment before merging them. For stream with more than 10 gaps, I simply skip them. Maybe these two parameters could be further moved into the default config. Other than that, this PR is really to go.

Cheers,
Yiyu

**Review by mdenolle — APPROVED** [review]
looks good!

File: src/noisepy/seis/pnwstore.py [hunk 1]
```diff
@@ -101,12 +101,16 @@ def read_data(self, timespan: DateTimeRange, chan: Channel) -> ChannelData:
             f"FROM tsindex WHERE network='{chan.station.network}' AND station='{chan.station.name}' "
             f"AND channel='{chan.type.name}' and location='{chan.station.location}' "
             f"AND filename LIKE '%%/{chan.station.network}/{year}/{doy}/%%'"
+            "ORDER BY byteoffset ASC"
```
Comment (carlosgjs) [inline src/noisepy/seis/pnwstore.py:104]: @niyiyu does this need a space at the end of the previous line? `.../%%' "`

Comment (niyiyu) [reply src/noisepy/seis/pnwstore.py:104]: Oh seems I missed the space here. However, they seem giving the same results : `.../%%'ORDER BY`..😂
Do you think it worth another PR to add this space?

**Pull request merged**